### ISSUE TITLE
Contact URL "Add to Calendar" has Timezone issue

### ISSUE THREAD

**Issue by rgbink** [pojome/elementor#6354]
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

Created two buttons and assigned a dynamic "Contact URL" action to them. One was Add to Outlook Calendar, the other Google. Both work as expected but on adding the event to my Google Calendar the time is off (early) by 5 hours. Event set for 8pm-1am following day, but is added to calendar as 3pm to 8pm starting day.

## Steps to reproduce

See description.

## Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Images
<img width="1408" alt="elementor___carnevale_2019_" src="https://user-images.githubusercontent.com/240772/49518282-3f7b4600-f86c-11e8-8839-d1b5d6b2ed44.png">
<img width="643" alt="google_calendar_-_event_details" src="https://user-images.githubusercontent.com/240772/49518598-042d4700-f86d-11e8-8922-24afdcd8d4e0.png">

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->

Server Environment |   |  
-- | -- | --
Operating System: | Linux |  
Software: | Apache/2.4.10 (Debian) |  
MySQL version: | MySQL Community Server (GPL) v5.6.34 |  
PHP Version: | 7.2.0 |  
PHP Max Input Vars: | 3000 |  
PHP Max Post Size: | 1000M |  
GD Installed: | Yes |  
ZIP Installed: | Yes |  
Write Permissions: | All right |  
Elementor Library: | Connected |  
</details>

**Comment by rgbink** [comment]
Finally tested out the Yahoo link (had to create an account) and there's date/time weirdness there as well. See attached…
<img width="574" alt="yahoo__calendar__december_2018" src="https://user-images.githubusercontent.com/240772/49546700-e7b1fe80-f8ae-11e8-82e9-0dd5baec5eb9.png">

**Comment by bainternet** [comment]
@rgbink 

Thanks for reporting, should be fixed next update.

**Comment by bainternet** [comment]
Thanks, fixed in Elementor pro v2.2.5
